common/src/main/webapp/sass/explore.scss

```diff
@@ -2014,6 +2014,11 @@ svg.fa-triangle-exclamation.explore-error-icon {
 // ------
 // Proposals and Partner Splits
 // ------
+
+.explore-abstract-too-long {
+  color: var(--error-background-color);
```

Comment:
There's a recently added `--negative-row-color` that you may want to use here instead of using a background color for the text. (There's also a `--negative-row-background`)

Reply:
I tried but couldn't find the new variable even after a rebase

Reply:
> I tried but couldn't find the new variable even after a rebase

It's defined in lucuma-ui since it's shared with observe.